packages/labs/eleventy-plugin-lit/package.json

```diff
@@ -29,14 +29,64 @@
     "node": ">=12.16.0"
   },
   "scripts": {
-    "build": "npm run clean && npm run build:ts",
-    "build:ts": "tsc --build",
-    "build:ts:watch": "tsc --build --watch",
-    "build:demo": "cd demo && NODE_OPTIONS=--experimental-vm-modules eleventy",
-    "build:demo:watch": "cd demo && NODE_OPTIONS=--experimental-vm-modules eleventy --watch",
-    "clean": "rimraf \"index.{js,js.map,d.ts,d.ts.map}\" \"test/**/*.{js,js.map,d.ts,d.ts.map}\" \"worker/**/*.{js,js.map,d.ts,d.ts.map}\" demo/_site *.tsbuildinfo",
-    "serve:demo": "wds --watch --node-resolve --open demo/_site/test",
-    "test": "uvu test/ \".*-test.js$\""
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:demo": "wireit",
+    "serve:demo": "wireit",
+    "test": "wireit"
```

Comment:
Not sure if I did something wrong. But I did a clean install of this branch.

`npm install` and `npm run bootstrap` at the monorepo root. Then I cd into `labs/eleventy-plugin-lit` and ran `npm run test`.

I got the following test result:

```
eleventy-plugin-test.js
• ✘ ✘ ✘ ✘ • ✘ ✘ ✘ • ✘ ✘ ✘ • ✘ ✘   (4 / 16)

   FAIL  "basic component in markdown file"
    Expected values to be deeply equal:  (equal)

        ++0    (Expected)
        --1    (Actual)
```

Did I hold it wrong?

Reply:
This may be related to the CI error: https://github.com/lit/lit/runs/6359879797?check_suite_focus=true#step:9:732

Which has a different error in ssr.

Reply:
Oops, thank you for checking that!

Yes I forgot to do a clean build check on that target. It needed to depend on the full build, not just typescript.